service-runner/src/main/resources/config/assets.yaml

```diff
@@ -14,6 +14,36 @@ assets:
       fee_fixed: 0
       fee_percent: 0
       max_amount: 1000000
+    sep6:
```

Comment:
I don't think we are validating assets.yaml at all. I can add validation though.

Reply:
I don't think we should define defaults because it's hard to define reasonable ones. Rather, we can have the validation catch any missing required fields.